client/src/app/(modules)/network/(main)/network.tsx

```diff
@@ -39,16 +46,27 @@ const Icons = ({
       region_of_interventions,
     } = (attributes as Project) || {};
     const projectType = project_type?.data?.attributes?.name;
+    const projectTypeDescription = project_type?.data?.attributes?.description;
     const regionName = isWorldwide
       ? 'Worldwide'
       : region_of_interventions?.data?.map((r) => r.attributes?.name).join(', ');
     return (
       <div className="flex flex-wrap gap-x-4 gap-y-2">
         {projectType && (
-          <div className="flex gap-2">
-            <FolderIcon className="h-6 w-6 min-w-min" />
-            <div className="text-base text-slate-500">{projectType}</div>
-          </div>
+          <TooltipProvider>
```

Comment:
If there is no description, we showing an empty tooltip. I think it would be better not to show any tooltip in this case.

Reply:
The tooltip on the regions below (see `WithEllipsis`) has a different style. Can you see with Lidia if we could harmonise the styles?

Reply:
Could we allow keyboard users to trigger the tooltip? I think it may be related to the `asChild` prop you set on `TooltipTrigger`.